lib/ansible/modules/cloud/ovirt/ovirt_vnic_profile.py

```diff
@@ -149,45 +149,62 @@ class EntityVnicPorfileModule(BaseModule):
     def __init__(self, *args, **kwargs):
         super(EntityVnicPorfileModule, self).__init__(*args, **kwargs)
 
-    def __get_dcs_service(self):
+    def _get_dcs_service(self):
         return self._connection.system_service().data_centers_service()
 
-    def __get_dcs_id(self):
-        return get_id_by_name(self.__get_dcs_service(), self.param('data_center'))
+    def _get_dcs_id(self):
+        return get_id_by_name(self._get_dcs_service(), self.param('data_center'))
 
-    def __get_network_id(self):
-        networks_service = self.__get_dcs_service().service(self.__get_dcs_id()).networks_service()
+    def _get_network_id(self):
+        networks_service = self._get_dcs_service().service(self._get_dcs_id()).networks_service()
         return get_id_by_name(networks_service, self.param('network'))
 
-    def __get_qos_id(self):
-        qoss_service = self.__get_dcs_service().service(self.__get_dcs_id()).qoss_service()
+    def _get_qos_id(self):
+        qoss_service = self._get_dcs_service().service(self._get_dcs_id()).qoss_service()
         return get_id_by_name(qoss_service, self.param('qos'))
 
-    def __get_network_filter_id(self):
+    def _get_network_filter_id(self):
         nf_service = self._connection.system_service().network_filters_service()
         return get_id_by_name(nf_service, self.param('network_filter')) if self.param('network_filter') else None
 
+    def _get_network_filter(self):
+        network_filter = None
+        if self.param('network_filter'):
+            network_filter = otypes.NetworkFilter(id=self._get_network_filter_id())
+        elif self.param('network_filter') == '' or self.param('pass_through') == 'enabled':
```

Comment:
@machacekondra 
should user remove the network_filter manually with the pass_through 'enabled' or keep it automatically

Reply:
Please document it, how it behaves.